internal/mempool/mempool.go

```diff
@@ -1903,6 +1898,24 @@ func (mp *TxPool) MaybeAcceptTransaction(tx *dcrutil.Tx, isNew, rateLimit bool)
 	if isAutoRevocationsEnabled {
 		checkTxFlags |= blockchain.AFAutoRevocationsEnabled
 	}
+	return checkTxFlags, nil
+}
+
+// MaybeAcceptTransaction is the main workhorse for handling insertion of new
+// free-standing transactions into a memory pool.  It includes functionality
+// such as rejecting duplicate transactions, ensuring transactions follow all
+// rules, orphan transaction handling, and insertion into the memory pool.  The
+// isOrphan parameter can be nil if the caller does not need to know whether
```

Comment:
`isOrphan` doesn't exist in this function.

Reply:
Good call.  It was already there, but since this is being seen as modified by git due to the refactor, I've gone ahead and removed that bit.